include/SSVOpenHexagon/Core/HexagonGame.hpp

```diff
@@ -474,8 +474,14 @@ class HexagonGame
 
     [[nodiscard]] float getMusicDMSyncFactor() const
     {
-        return Config::getMusicSpeedDMSync() ? std::pow(difficultyMult, 0.12f)
-                                             : 1.f;
+        return levelStatus.syncMusicToDM ? std::pow(difficultyMult, 0.12f)
+                                         : 1.f;
+    }
+
+    void setMusicPitch(sf::Music* current)
```

Comment:
If you are expecting `current` to never be `nullptr`, it is recommended to accept it by reference to avoid introducing the extra null state in the type system:

```suggestion
    void setMusicPitch(sf::Music& current)
```

Reply:
Ah, I see. This was the original case of how the auto was defined in a code block, and I guess this was a mistake in the code originally. I'll commit to your suggestion.